index.html

```diff
@@ -702,80 +703,100 @@ <h2>
 
       <p>
 The generic <a>DID scheme</a> is a URI scheme conformant with
-[[RFC3986]]. It consists of a DID followed by an optional path and/or
-fragment. The term DID refers only to the identifier conforming to
-the did rule in the ABNF below; when used alone, it does not include
-a path or fragment. A DID that may optionally include a path and/or
-fragment is called a DID reference.
+[[RFC3986]]. It consists of a DID method name, followed by a method-specific
+identifier. The term DID refers only to the identifier conforming to
+the did rule in the ABNF below.
+
+Additionally, a DID can be used as a component in a DID Reference, described in
+the <a href="#did-reference">DID Reference section</a> below.
       </p>
 
       <p>
-Following is the ABNF definition using the syntax in [[RFC5234]]
-        (which defines ALPHA as upper or lowercase A-Z).
+The following is the ABNF definition using the syntax in [[RFC5234]]
+which defines <code>ALPHA</code> and <code>DIGIT</code>.
       </p>
 
       <pre class="nohighlight">
-did-reference      = did [ "/" did-path ] [ "#" did-fragment ]
-did                = "did:" method ":" specific-idstring
+did                = "did:" method ":" method-specific-idstring
 method             = 1*methodchar
 methodchar         = %x61-7A / DIGIT
-specific-idstring  = idstring *( ":" idstring )
+method-specific-idstring  = idstring *( ":" idstring )
 idstring           = 1*idchar
 idchar             = ALPHA / DIGIT / "." / "-"
 </pre>
-      <p>
-See Sections <a href="#paths"></a> and <a href="#fragments"></a> for the ABNF rules defining DID paths and
-fragments.
-      </p>
     </section>
 
     <section>
       <h2>
-Paths
+DID Reference
       </h2>
 
       <p>
-A generic <a>DID path</a> (the did-path rule in Section <a href="#the-generic-did-scheme"></a>) is identical to a URI path and MUST
-conform to the ABNF of the path-rootless ABNF rule in [[RFC3986]]. A
-DID path SHOULD be used to address resources available via a DID
-service endpoint. See Section <a href="#service-endpoints"></a>.
+DID References are used for two main purposes: to refer to sections
+inside a DID Document (via <a href="#did-fragments">DID Fragments</a>), and as a
+portable way to refer to Service Endpoint resources, queries and fragments.
```

Comment:
oxford comma and link:
```suggestion
portable way to refer to <a>Service Endpoint</a> resources, queries, and fragments.
```